submission_runner.py

```diff
@@ -196,12 +196,20 @@ def train_once(
 
   # Workload setup.
   logging.info('Initializing dataset.')
+  use_mixup = (
+      hyperparameters.use_mixup
+      if hasattr(hyperparameters, 'use_mixup') else False)
+  mixup_alpha = (
+      hyperparameters.mixup_alpha
+      if hasattr(hyperparameters, 'mixup_alpha') else 0.1)
```

Comment:
we will never tune these, because the competition rules prohibit data aug changing on a per-workload basis, so I think we can remove all the changes to submision_runner.py